init/retina/main_linux.go

```diff
@@ -15,9 +18,20 @@ import (
 func main() {
 	// Initialize logger
 	opts := log.GetDefaultLogOpts()
+	zl, err := log.SetupZapLogger(opts)
+	if err != nil {
+		panic(err)
```

Comment:
We're in a good position to do this refactor without too much pain:

```go
func main() {
       if err := run(); err != nil {
              fmt.Println(err)
              os.Exit(1)
       }
}

func run() error {
       // snip
       zl, err := log.SetupZapLogger(opts)
       if err != nil {
              return fmt.Errorf("setting up zap logger: %w", err)
       }
}
```

That way we don't have to panic. For bonus points, you can even eliminate the global dep on `os.Args` this way:

```go
func main() {
       if err := run(os.Args[1:]...); err != nil {
              fmt.Println(err)
              os.Exit(1)
       }
}

func run(args ...string) error {
       // snip
       zl, err := log.SetupZapLogger(opts)
       if err != nil {
              return fmt.Errorf("setting up zap logger: %w", err)
       }

       fs := NewFlagSet("retina", flag.ContinueOnError)
       fs.String("config", "/retina/config/config.yaml", "path to the config file")
       // snip
       err := fs.Parse(args)
       if err != nil {
              return fmt.Errorf("parsing flags: %w", err)
       }
}
```